ui/imports/shared/stores/send/TransactionStore.qml

```diff
@@ -58,14 +58,14 @@ QtObject {
         return networksModule.getBlockExplorerTxURL(chainID)
     }
 
-    function authenticateAndTransfer(uuid, slippagePercentage = "") {
-        walletSectionSendInst.authenticateAndTransfer(uuid, slippagePercentage)
+    function authenticateAndTransfer(uuid) {
+        walletSectionSendInst.authenticateAndTransfer(uuid)
     }
 
-    function suggestedRoutes(uuid, amountIn, amountOut = "0", extraParamsJson = "") {
+    function suggestedRoutes(uuid, amountIn, amountOut = "0", slippagePercentage = "", extraParamsJson = "") {
```

Comment:
```suggestion
    function suggestedRoutes(uuid, amountIn, amountOut = "0", slippagePercentage = 0.0, extraParamsJson = "") {
```

Reply:
@caybro but expected type on the Nim side is string, and then we parse it.
Check here:
https://github.com/status-im/status-desktop/blob/aa03be96af06628a2e6092ab27c4fb638b946c6e/src/app/modules/main/wallet_section/send_new/view.nim#L32-L45

Reply:
I see... it's just that in swap adaptor/form, it's declared as a `double`